presto-parser/src/main/java/com/facebook/presto/sql/ExpressionFormatter.java

```diff
@@ -705,7 +711,7 @@ else if (codePoint <= 0xFFFF) {
 
     static String formatOrderBy(OrderBy orderBy, Optional<List<Expression>> parameters)
     {
-        return "ORDER BY " + formatSortItems(orderBy.getSortItems(), parameters);
+        return " ORDER BY " + formatSortItems(orderBy.getSortItems(), parameters);
```

Comment:
I think other methods assume that returned strings don't have trailing spaces. Maybe callers should adjust to that convention?